modules/rest-api/pages/rest-xdcr-adv-settings.adoc

```diff
@@ -345,6 +425,17 @@ For information, see the xref:xdcr-reference:xdcr-advanced-settings.adoc#table-o
 
 This setting can be established and retrieved either for an individual replication or globally.
 
+| `conflictLogging`
+| JSON Object
+| Configuration settings for logging conflicts.
+
+This configuration setting defines objects/parameters and options used to control how conflicts are logged during an XDCR replication.
+
+Within the JSON Object, `conflictLogging` setting is enabled by default (`disabled=false`).
```

Comment:
This is incorrect.  Conflict logging is disabled by default.  
"conflictLogging": {},

Should just remove line 434 since it's confusing.